src/pfs/pfs.proto

```diff
@@ -462,6 +462,51 @@ message RunLoadTestResponse {
   google.protobuf.Duration duration = 5;
 }
 
+message ObjectStorageEgressOptions {
+  string url = 1;
+}
+
+message SQLEgressOptions {
+  message FileFormat {
+    enum Type {
+        UNKNOWN = 0;
+        CSV = 1;
+        JSON = 2;
+        PARQUET = 3;
+    }
+    message Options {
+      repeated string json_field_names = 1;
+    }
+    Type type = 1;
+    Options options = 2;
+  }
+  message Secret {
+    string k8s_secret = 1;
+    string key = 2;
+    string mount_path = 3;
+    string env_var = 4;
+  }
```

Comment:
`mount_path` is ignored, right? and I'd rather we not give people the option to set `env_var` if there's only one working value for it, unless the plan is to unify this with the `pps` type when we get a chance